daemon/vpn/nordlynx/libtelio/libtelio.go

```diff
@@ -538,6 +496,19 @@ func (l *Libtelio) StatusMap() (map[string]string, error) {
 	return m, nil
 }
 
+func nodeStateToString(state teliogo.NodeState) string {
+	switch state {
+	case teliogo.NodeStateConnected:
+		return "connected"
+	case teliogo.NodeStateConnecting:
+		return "connecting"
+	case teliogo.NodeStateDisconnected:
+		return "disconnected"
+	default:
+		panic(fmt.Sprintf("unexpected telio.NodeState: %#v", state))
```

Comment:
Maybe we shouldn't `panic` if we cannot convert the state to string? Instead just return unknown/disconnected or empty string and log an error message?

Reply:
Yes, definitely. It was supposed to be only a placeholder. I missed it for some reason. Fixed. Good catch :+1: